src/sites/link/urlbluemedia.shop.js

```diff
@@ -0,0 +1,10 @@
+_.register({
+  rule: {
+    host: /^urlbluemedia.shop$/,
```

Comment:
please fix the code, should be:
`    host: /^urlbluemedia\.shop$/,`

Reply:
Fixed